security/src/main/java/com/networknt/security/JwtVerifier.java

```diff
@@ -83,13 +83,20 @@ public class JwtVerifier {
     public static final String JWT_KEY_RESOLVER_X509CERT = "X509Certificate";
     public static final String JWT_KEY_RESOLVER_JWKS = "JsonWebKeySet";
 
+    public static final String ENABLE_JWT_SIGNATURE_VALIDATION = "skipSignatureCheck";
+
+    public static final String IGNORE_EXPIRY ="ignoreExpiry";
 
     Map<String, Object> config;
     Map<String, Object> jwtConfig;
     int secondsOfAllowedClockSkew;
     Boolean enableJwtCache;
     Boolean bootstrapFromKeyService;
 
```

Comment:
Please remove unneccessary blank lines. Should be 

    ...
    Boolean enableJwtCache;
    Boolean bootstrapFromKeyService;
    Boolean skipSignatureCheck;
    Boolean ignoreExpiry;

    static Cache<String, JwtClaims> cache;
    ...